packages/protocol/script/layer1/based/DeployProtocolOnL1.s.sol

```diff
@@ -406,24 +450,19 @@ contract DeployProtocolOnL1 is DeployCapability {
             registerTo: resolver
         });
 
-        router = deployProxy({
-            name: "preconf_router",
-            impl: address(new PreconfRouter(taikoWrapper, whitelist)),
-            data: abi.encodeCall(PreconfRouter.init, (owner)),
-            registerTo: resolver
-        });
-
+        // Initializable a forced inclusion store with a fake address for TaikoWrapper at first,
+        // to be used for deploying TaikoWrapper, then upgrade it to the real TaikoWrapper address.
         store = deployProxy({
             name: "forced_inclusion_store",
             impl: address(
                 new ForcedInclusionStore(
                     uint8(vm.envUint("INCLUSION_WINDOW")),
                     uint64(vm.envUint("INCLUSION_FEE_IN_GWEI")),
                     taikoInbox,
-                    taikoWrapper
+                    address(1)
```

Comment:
Since there is a cycle dependency for `ForcedInclusionStore.constructor` and `TaikoWrapper.constructor`, so have to create a store with fake taikoWrapper address at first here. @dantaik @xiaodino @YoGhurt111